app/scripts/views/app.coffee

```diff
@@ -569,10 +564,22 @@ define [
       # TODO @jrwdunham: use the already-in-place @alertDialog for this
       # TODO @jrwdunham @cesine: figure out how i18n/localization works in
       # FieldDB and begin implementing something similar in Dative.
-      console.log ["TODO show some visual thing here using the app view using",
+      console.log ["Pretending the user said Yes. This is dangerous. TODO show some visual confirm dialog here using the app view using",
         "something like http://jqueryui.com/dialog/#modal-confirmation" ,
         message].join ' '
```

Comment:
Dative has a modal confirm widget that can be used for this. It's the `AlertDialogView`, I think. See an example of its usage in the `FormView` when the delete/destroy button is clicked.